client/assets/images/illustrations/jetpack-backup.svg

```diff
@@ -1 +1,36 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="116" height="120"><defs><clipPath clipPathUnits="userSpaceOnUse" id="a"><path d="M92.62 47.86C89.79 33.13 77.17 22.08 62 22.08c-12.04 0-22.5 7-27.71 17.24C21.75 40.69 12 51.57 12 64.76c0 14.13 11.21 25.62 25 25.62h54.17c11.5 0 20.83-9.57 20.83-21.35 0-11.27-8.54-20.4-19.38-21.17z"/></clipPath><clipPath clipPathUnits="userSpaceOnUse" id="b"><path d="M57.75 65.93L45.93 56.1l-3.2 4.03 15.76 13.11 19.2-24.23-3.94-3.28-16 20.2z"/></clipPath></defs><g clip-path="url(#a)"><path d="M12 5h100v102.45H12V5z" fill="#069e08"/></g><path fill-rule="evenodd" d="M57.75 65.93L45.93 56.1l-3.2 4.03 15.76 13.11 19.2-24.23-3.94-3.28-16 20.2z" fill="#d8d8d8"/><g clip-path="url(#b)"><path fill="#fff" d="M38.43 41.21l40.4-4.31 4.2 41.38-40.4 4.31-4.2-41.38z"/></g><path fill-rule="evenodd" d="M26 24.57C26 22.6 27.6 21 29.57 21c1.97 0 3.57 1.6 3.57 3.57 0 1.97-1.6 3.57-3.57 3.57-1.97 0-3.57-1.6-3.57-3.57z" fill="#d0e6b8"/><path fill-rule="evenodd" fill="#9dd977" d="M16.12 6.9L17.8 3l-3.9 1.68L10 3l1.68 3.9L10 10.8l3.9-1.68 3.9 1.68-1.68-3.9zm98 25l1.68-3.9-3.9 1.68L108 28l1.68 3.9-1.68 3.9 3.9-1.68 3.9 1.68-1.68-3.9zm-12.46-13.24l2.6-1.03-2.6-1.03-1.03-2.6-1.03 2.6-2.6 1.03 2.6 1.03 1.03 2.6 1.03-2.6zm-97 23l2.6-1.03-2.6-1.03L3.63 37 2.6 39.6 0 40.63l2.6 1.03 1.03 2.6 1.03-2.6z"/><path d="M62 118c-21.57 0-39-1.79-39-4s17.43-4 39-4 39 1.79 39 4-17.43 4-39 4z" fill="#f2f2f3"/></svg>
```

Comment:
Better if we use the minified version

Reply:
Yup, during a rebase the prettified version sticked. I'll revert that.